applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -492,7 +524,18 @@ static void discover_sink_cb(struct bt_conn *conn, struct bt_codec *codec, struc
 	}
 
 	if (codec != NULL) {
-		add_remote_codec(codec, params->num_caps, params->dir);
+		if (codec->id != BT_CODEC_LC3_ID) {
+			LOG_DBG("Only the LC3 codec is supported");
+			return;
+		}
+
+		if (channel_index < ARRAY_SIZE(temp_codec_cap)) {
```

Comment:
Wouldn't the channel_index always be 0 zero here, since it is set on line:518 and it's not static

Reply:
Changed from channel_index to params->num_caps